api/src/main/java/com/example/api/model/map/requirement/RequirementValueType.java

```diff
@@ -0,0 +1,17 @@
+package com.example.api.model.map.requirement;
+
+public enum RequirementValueType {
+    DATE("date"),
+    NUMBER("number"),
+    MULTI_SELECT("multi_select");
+
```

Comment:
Zastanawiam się czy mając do wyboru te 3 wartości będzie można odróżnić w responsach np. date_to od date_from. W przykładowym responsie requirementow były ustawione odpowiednie nazwy, ale co będzie gdy te nazwa nie będzie tak oczywista (zakładam, że nazwę requirementów można dowolnie ustawić)

Reply:
To już w tym pr robiłem. Z frontu dostajesz requirementId, pobieram z niego requirement, a requirement, kazdy ma RequirementType, dzięki któremu odróżnie dateFrom od dateTo. RequirementValueType jest tylko po to, bo front tego potrzebuje i wysyłam to im